ModpackDownloader.sln

```diff
@@ -7,38 +6,54 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ModpackDownloader", "Modpac
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ApplicationEssentials", "ApplicationEssentials\ApplicationEssentials\ApplicationEssentials.csproj", "{92D0B219-FBDF-434C-87F1-51B8C893F86D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ModpackDownloader.Tests", "ModpackDownloader.Tests\ModpackDownloader.Tests.csproj", "{EFAA486F-845F-48D6-B15B-21E44B541737}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ModpackDownloader.Tests", "ModpackDownloader.Tests\ModpackDownloader.Tests.csproj", "{EFAA486F-845F-48D6-B15B-21E44B541737}"
```

Comment:
There is nothing new here, just VS being VS. Please remove it from the pull request.